dotCMS/src/main/java/com/dotcms/rest/WebResource.java

```diff
@@ -760,6 +763,7 @@ private UsernamePassword(String username, String password) {
         private String[] requiredPortlet = null;
         private final Set<String> requiredRolesSet = new HashSet<>();
         private AnonymousAccess anonAccess=AnonymousAccess.NONE;
+        private boolean requireLicense = false;
```

Comment:
Codacy found an issue: [Avoid using redundant field initializer for 'requireLicense'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5883793)